profiles/common/modules/features/communities/communities.module

```diff
@@ -456,10 +456,15 @@ function communities_views_post_execute(&$view) {
   if (isset($view->argument['gid'])) {
     $handler = $view->argument['gid'];
     if ($handler->table == 'og_membership' && $handler->field == 'gid') {
-      // Postiton 0 = argument 1 (%1, !1)...
+      // Position 0 = argument 1 (%1, !1)...
       $position = $handler->position + 1;
       // Not use node_load for performance reason.
-      $title = db_query('SELECT title FROM {node} WHERE nid = :nid', array(':nid' => $handler->argument))->fetchField();
+      $title = db_select('node', 'n')
+        ->fields('n', array('title'))
+        ->condition('nid', $handler->argument, '=')
+        ->addTag('node_access')
```

Comment:
It is not a blocking comment as others modules (contrib and custom) use already 'node_access' as tag but I strongly recommend to follow it.

Instead of 'node_access', I would use 'communities_node_access'. It is safer than 'node_access' that is used in other modules and then, we could have strange effects if a module implement the 'hook_query_alter()' to modify a query tagged by 'node_access'.